lib/kubeclient_next/kubeconfig/cluster.rb

```diff
@@ -3,24 +3,26 @@
 module KubeclientNext
   module Kubeconfig
     class Cluster
-      attr_reader :name, :insecure_skip_tls_verify, :certificate_authority, :server
+      attr_reader :name, :insecure_skip_tls_verify, :certificate_authority, :certificate_authority_data, :server
```

Comment:
This may not yet be an exhaustive list (here and elsewhere). Once a baseline level of functionality exists, it will be easier to start filling in any holes when proving against different kinds of real clusters. For now I just have a local KinD cluster (hence the cert-based auth)